packages/react/src/index.ts

```diff
@@ -26,4 +26,5 @@ export * from './tabs';
 export * from './toggle';
 export * from './toggle-group';
 export * from './tooltip';
+export * from './use-render';
```

Comment:
Why `useRender` is not inside `/utils`?

Reply:
@michaldudak @atomiks what are your thoughts? I though it as a "first class export" similar to the components, but having it in the utils makes sense as well, especially if we want to publicly make available more of the hooks.

Reply:
We decided that it's best to export from a top level directory.